src/app/components/OnChainReceive/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { CaretLeftIcon } from "@bitcoin-design/bitcoin-icons-react/filled";
+import Header from "@components/Header";
+import { Trans, useTranslation } from "react-i18next";
+import { useNavigate } from "react-router-dom";
+import Button from "~/app/components/Button";
+import Container from "~/app/components/Container";
+import IconButton from "~/app/components/IconButton";
+
+export default function OnChainReceive() {
+  const navigate = useNavigate();
+  const { t } = useTranslation("translation", {
+    keyPrefix: "on_chain",
+  });
+
+  return (
+    <div className=" flex flex-col overflow-y-auto no-scrollbar">
+      <Header
+        title={t("title")}
+        headerLeft={
+          <IconButton
+            onClick={() => {
+              navigate(-1);
+            }}
+            icon={<CaretLeftIcon className="w-4 h-4" />}
+          />
+        }
+      />
+      <div className="pt-4">
+        <Container justifyBetween maxWidth="sm">
+          <div className="text-center dark:text-neutral-200 h-full flex flex-col justify-center items-center">
+            <div className="mb-8">
+              <p>
+                <Trans
+                  i18nKey={"instructions1"}
+                  t={t}
+                  components={[<strong key="instruction1-strong"></strong>]}
+                />
+              </p>
+            </div>
+
+            <div className="mb-8">
+              <p>
+                <Trans
+                  i18nKey={"instructions2"}
+                  t={t}
+                  components={[<strong key="instruction2-strong"></strong>]}
+                />
+              </p>
+            </div>
+          </div>
+          <div className="mb-4">
+            <a href="https://getalby.com/node/receive">
+              <Button
+                type="submit"
+                label={t("go")}
+                fullWidth
+                primary
+                onClick={() =>
+                  window.open(`https://getalby.com/user`, "_blank")
```

Comment:
Why is this button wrapped in a link and also has an `onClick` function? The onClick one has the wrong url too.

Reply:
this is from on chain Tabs pull request. i thought it is already reviewed and considered valid.
what shall be the behaviour here?

Reply:
changed to single url. /node/receive